python/.env.example

```diff
@@ -1,5 +1,6 @@
-OPENAI_API_KEY=""
-OPENAI_ORG_ID=""
-AZURE_OPENAI_DEPLOYMENT_NAME=""
-AZURE_OPENAI_ENDPOINT=""
-AZURE_OPENAI_API_KEY=""
+# Env vars use the same names too.
```

Comment:
Does this need to change? Not sure why.

Reply:
Hey. Apologies. I should have mentioned that in the commit message. It's for 2 reasons:
- Pydantic: The main reason is because of the way Pydantic works. Without this change, Pydantic is unable to load from `.env` files. We could add some custom functionality to make Pydantic work with `.env` files too, but that is additional code that needs to be maintained.
- Consistency: This is how the env vars are named.